### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/actorconditions.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#196]
```
translated actorconditions_v069_bwm
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 24 May 2011 at 9:09

Attachments:
- [actorconditions.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-196/comment-0/actorconditions.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Committed in r99. Thanks liogiu!
```

Original comment by `oskar.wi...@gmail.com` on 25 May 2011 at 7:56
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:53
- Changed state: **Done**
